WeWatch/WeWatch/Preview Content/Assets/Font/Font extension.swift

```diff
@@ -15,12 +15,27 @@ extension Font {
     internal static var poppinsBold24px: Font {
```

Comment:
Rename this file. All files containing extensions should follow the naming convention like: `Font+Extensions`.